### PR TITLE
Enable support for multiple interfaces

### LINKED ISSUE #32: Make multiple networks on multiple radios work
Our APs have two radios: One for 2.4GHz and one for 5GHz. 
We want these APs to be able to send out their network on both frequencies and work with wimoved. 

- [x] Explore what currently works: Can we combine all events into one hostAPD socket?
- [x] Decide on what needs to be done
- [x] Implement changes

### PR DESCRIPTION
**Pull request by dasGoogle**
Closes #32 

**Important**
You need to update your configuration file. Remove the line for hapd_sock. 
Instead, update as follows: 
```text
hapd_sockdir=/var/run/hostapd/
sockets=wlp1s0
```

### CODE REVIEW

**Comment by dasGoogle** [comment]
@Sohn123 @linaScience @rgwohlbold When testing, please from now on make sure to enable the hostapd global control interface, i.e. using
```bash
sudo hostapd ./hapd.conf -g /var/run/hostapd/global
```

File: src/Station.cpp [hunk 1]
```diff
@@ -8,11 +8,12 @@
 
 const auto MAC_REGEX = std::regex("^([0-9a-fA-F]{2}:){5}[0-9a-fA-F]{2}$");
 
-Station::Station(std::string mac) : vlan_id(std::nullopt) {
+Station::Station(std::string sockname, std::string mac) : vlan_id(std::nullopt) {
```
Comment (rgwohlbold) [inline src/Station.cpp:11]: We should use member initialization (`: sockname(sockname), mac(mac)`) here. `this->sockname = sockname` is not very idiomatic in C++

File: src/ipc/Caller.cpp [hunk 2]
```diff
@@ -5,42 +5,53 @@
...
+    Socket &socket = get_socket(station.sockname);
+    std::istringstream stream(socket.send_and_receive({"STA", station.mac}));
+    std::string line;
+    while (std::getline(stream, line)) {
+        if (line.rfind(VLAN_ID_PREFIX) == 0) {
+            return std::stol(line.substr(VLAN_ID_PREFIX.size(), line.size()));
         }
-        std::this_thread::sleep_for(std::chrono::milliseconds(100));
     }
+    std::this_thread::sleep_for(std::chrono::milliseconds(100));
```
Comment (rgwohlbold) [inline src/ipc/Caller.cpp:25]: We should remove the `sleep()` here

**Review by rgwohlbold — CHANGES_REQUESTED** [review]
Seems to work on my machine. I left a few nit-picky comments around the code for you to review

**Comment by dasGoogle** [comment]
I have deployed the current version of this PR on our APs at the office. Appears to work quite well but we should probably test roaming some more tomorrow when we have more devices available.

**Review by rgwohlbold — APPROVED** [review]
Looks good now! :shipit:

**Pull request merged**